mempool/cat/pool.go

```diff
@@ -91,8 +91,8 @@ func NewTxPool(
 		config:           cfg,
 		proxyAppConn:     proxyAppConn,
 		metrics:          mempool.NopMetrics(),
-		rejectedTxCache:  mempool.NopTxCache{},
-		evictedTxCache:   mempool.NopTxCache{},
+		rejectedTxCache:  mempool.NewRejectedTxCache(cfg.CacheSize),
```

Comment:
NopTxCache assignment was removed from on main for cat pool. I'm gonna do the same for priority mempool